src/image-sync-manager.ts

```diff
@@ -24,25 +23,29 @@ export class ImageSyncManager {
 
   private _syncEvents = () => {
     this._eventManager.listen(this._mainPlayer, this._mainPlayer.Event.TIMED_METADATA, this._onTimedMetadata);
-    this._eventManager.listen(this._mainPlayer, TIMED_METADATA_ADDED, this._onTimedMetadataAdded);
+    this._eventManager.listen(this._mainPlayer, this._mainPlayer.cuePointManager.TIMED_METADATA_ADDED, this._onTimedMetadataAdded);
   };
 
-  private _onTimedMetadata = ({payload}: {payload: {cues: Array<{track: {label: string; language: string}; value: {data: {id: string}}}>}}) => {
-    if (payload.cues[0]?.track?.label === 'KalturaCuePoints' && payload.cues[0]?.track?.language === SLIDE_CUE_POINT_TYPE) {
+  private _onTimedMetadata = ({payload}: {payload: {cues: Array<{track: {label: string}; value: {data: {id: string; cuePointType: string}}}>}}) => {
+    if (
+      payload.cues[0]?.track?.label === this._mainPlayer.cuePointManager.KalturaCuePointsTextTrack &&
```

Comment:
`KalturaCuePointsTextTrack` maybe remove kaltura as the cue point could be external